stdlib/hash.gr

```diff
@@ -227,13 +227,17 @@ let rec hashOne = (val, depth) => {
  *
  * @since v0.1.0
  */
-export let hash = (anything) => {
+export let rec hash = (anything) => {
```

Comment:
Looks like we should also be decRefing `anything`, right?

Reply:
Right.